shared/message/support/nusight/Command.proto

```diff
@@ -21,6 +21,13 @@ syntax = "proto3";
 
 package message.support.nusight;
 
+/**
+ * Command string.
+ */
 message Command {
+
+    /**
+     * Command string.
+     */
```

Comment:
You said this already

Reply:
I had no idea what to say for this one lol. You're right, that doesn't add anything. I have removed that comment for now.